community/kernel/src/test/java/org/neo4j/kernel/impl/api/KernelTransactionImplementationTest.java

```diff
@@ -458,11 +461,11 @@ public void markForTerminationTerminatedTransaction()
 
         KernelTransactionImplementation transaction = newInitializedTransaction( true, locks );
 
-        transaction.markForTermination();
-        transaction.markForTermination();
-        transaction.markForTermination();
+        transaction.markForTermination( Status.General.UnknownFailure );
+        transaction.markForTermination( Status.General.UnknownFailure );
+        transaction.markForTermination( Status.General.UnknownFailure );
```

Comment:
You could additionally verify that the first reason isn't overwritten by later reasons.